runtime/src/bank/builtin_programs.rs

```diff
@@ -62,3 +62,458 @@ mod tests {
         bank.finish_init(&genesis_config, None, false);
     }
 }
+
+#[cfg(test)]
+mod tests_core_bpf_migration {
+    use {
+        crate::bank::{
+            builtins::{
+                core_bpf_migration::{tests::TestContext, CoreBpfMigrationConfig},
+                BuiltinPrototype, StatelessBuiltinPrototype, BUILTINS, STATELESS_BUILTINS,
+            },
+            test_utils::goto_end_of_slot,
+            tests::{create_genesis_config, new_bank_from_parent_with_bank_forks},
+            Bank,
+        },
+        solana_sdk::{
+            account::{AccountSharedData, ReadableAccount, WritableAccount},
+            bpf_loader_upgradeable::{self, get_program_data_address, UpgradeableLoaderState},
+            feature::{self, Feature},
+            feature_set::FeatureSet,
+            instruction::Instruction,
+            message::Message,
+            native_loader,
+            native_token::LAMPORTS_PER_SOL,
+            pubkey::Pubkey,
+            signature::Signer,
+            transaction::Transaction,
+        },
+        std::sync::Arc,
+        test_case::test_case,
+    };
+
+    const TEST_ELF: &[u8] =
+        include_bytes!("../../../programs/bpf_loader/test_elfs/out/noop_aligned.so");
```

Comment:
This bakes the entire ELF into the tests executable. Usually we load these at runtime.

Reply:
Ahh good point. I'll have to change this in this spot, as well as the other tests in my previous PR. Should be minimal and I'll add a commit here for it.

Reply:
All set!